### PR TITLE
Add strings for Command Queue translation

### PR DESCRIPTION
**Pull request by Philoul**

### CODE REVIEW

**Comment by MilosKozak** [comment]
tests needs to be fixed (mock new strings)

**Comment by Philoul** [comment]
After several hours 🤯 I finally find fixes for tests.
Not sure it's the best solution even if tests pass.
Please tell me if it's ok or if there is a better solution.

File: app/src/test/java/info/nightscout/androidaps/queue/QueueThreadTest.kt [hunk 1]
```diff
@@ -41,6 +43,7 @@ class QueueThreadTest : TestBaseWithProfile() {
             }
             if (it is CommandTempBasalAbsolute) {
                 it.activePlugin = activePlugin
+                it.rh = rh
```
Comment (MilosKozak) [inline app/src/test/java/info/nightscout/androidaps/queue/QueueThreadTest.kt:46]: this should be covered by `if (it is Command)` or?

**Comment by Philoul** [comment]
Oups, Crash detected in CommandTempBasalPercent (strange, it seems to be only during Disconnect Pump)

**Pull request merged**